src/components/Explore/TokenDetails/FooterBalanceSummary.tsx

```diff
@@ -0,0 +1,183 @@
+import { useToken } from 'hooks/Tokens'
+import { useNetworkTokenBalances } from 'hooks/useNetworkTokenBalances'
+import { useState } from 'react'
+import { AlertTriangle } from 'react-feather'
+import styled from 'styled-components/macro'
+
+import { SMALLEST_MOBILE_MEDIA_BREAKPOINT } from '../constants'
+import { LoadingBubble } from '../loading'
+
+const FAKE_NAV_FOOTER_HEIGHT = '56px'
```

Comment:
Is this meant to be in the codebase?

Reply:
@kayleegeorge can you update the naming of this?

Reply:
if the new nav bar is going in with this PR I can take this out, but else I'll just name it "placeholder"? @vm ?

Reply:
its not. we can leave this for now then